src/domains/ValuesDomain.js

```diff
@@ -259,6 +259,52 @@ export default class ValuesDomain {
     invariant(false, "unimplemented " + op);
   }
 
+  static logicalOp(realm: Realm, op: BabelNodeLogicalOperator, left: ValuesDomain, right: ValuesDomain): ValuesDomain {
```

Comment:
As this calls `computeLogical`, this function deserves a similar comment as computeLogical regarding side effects.

Reply:
This is not supposed to have side-effects.